README.md

```diff
@@ -75,7 +77,7 @@ scripts:
     build:
         default: webpack
         prod: webpack -p
-    validate: nps --parallel lint,test,build
+    validate: concurrently "nps lint" "nps test" "nps build"
```

Comment:
Concurrently might prove [an extra headache for Cygwin users](https://github.com/kimmobrunfeldt/concurrently/issues/69#issuecomment-257698185), I guess. I'll try to see if there's any way to fix that though instead of using that as a basis for the decision at hand.